nimare/annotate/gclda.py

```diff
@@ -663,28 +645,61 @@ def _update_regions(self):
 
                 # Estimate Covariances
                 # Covariances are estimated independently
```

Comment:
@tyarkoni, this means that, while the centroids of the Gaussians will be symmetric, the actual shapes could be different, right? I've noticed that in fits in the past and it only just struck me recently that that was different from what I assumed would happen (i.e., I assumed that the symmetric distributions would be completely symmetric).